lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -936,6 +936,14 @@ mapping:
           - $iso8601 (complete format string as specified by ISO 8601 international
             standard).
 
+      user_preferences_extra_conf_path:
+        type: str
+        default: 'config/user_preferences_extra_conf.yml'
+        required: false
+        desc: |
+          Location of the user_preferences_extra_conf.yml file, if not in the
+          standard configuration directory.
```

Comment:
I was wondering if here we should replace `user_preferences_extra_conf.yml` with a description, e.g.: `Location of the configuration file containing extra user preferences.`

And I think we can drop `if not in the standard configuration directory.`

Reply:
sounds good